core/src/main/scala/org/http4s/HttpService.scala

```diff
@@ -0,0 +1,31 @@
+package org.http4s
+
+import cats._
+import cats.data._
+import org.http4s.syntax.kleisli._
+
+object HttpService extends Serializable {
+
+  /**
+    * Lifts a total function to an `HttpService`. The function is expected to
+    * handle all requests it is given.  If `f` is a `PartialFunction`, use
+    * `apply` instead.
+    */
+  def lift[F[_]: Functor](f: Request[F] => F[Response[F]]): HttpService[F] =
+    Kleisli(f).liftOptionT
+
+  def liftF[F[_]](f: Request[F] => OptionT[F, Response[F]]): HttpService[F] =
```

Comment:
We call it `liftF` because it <em>lift</em>s a <em>F</em>unction?  I don't have a better name.  Cats calls this `apply`, but that's already taken.  `lift` is confusing for the existing `lift` and for this, because `Kleisli.lift` takes just the `F[B]`.

Does this help with type inference, or could we even deprecate this one and just use Kleisli for things that are already Kleisli functions and leave our `apply` as the DSL for partiality?